translate/public/locale/en-US/translate.ftl

```diff
@@ -503,7 +503,7 @@ machinery-ConcordanceSearch--translation-memory = TRANSLATION MEMORY
 
 ## Machinery Google Translation
 ## Shows the translation source from Google Translate.
-machinery-GoogleTranslation--visit-google = GOOGLE TRANSLATE
+machinery-GoogleTranslation--translation-source = GOOGLE TRANSLATE
     .title = Visit Google Translate
```

Comment:
This is the bit that's incorrect now that it's no longer a link.
```suggestion
```

Reply:
As discussed in our call, we are keeping this for now.